tools/build_h3_tools.sh

```diff
@@ -169,3 +169,38 @@ autoreconf -i
   LDFLAGS="${LDFLAGS}"
 ${MAKE} -j $(nproc)
 sudo ${MAKE} install
+
+# boringssl
+echo "Building boringssl..."
+
+# We need this go version.
+GO_BASE_PATH=/usr/local
+if [ ! -d ${GO_BASE_PATH} ]; then
+  sudo mkdir ${GO_BASE_PATH}
+fi
+
+wget https://go.dev/dl/go1.20.1.linux-amd64.tar.gz
+sudo rm -rf ${GO_BASE_PATH}/go && sudo tar -C ${GO_BASE_PATH} -xf go1.20.1.linux-amd64.tar.gz
+rm go1.20.1.linux-amd64.tar.gz
```

Comment:
Quite a few of us build in docker containers on m1 macs. Let's conditionally use `arm64` based upon `uname -m`.

I think this can change to:

```sh
if [ `uname -m` = "arm64" ]; then
    ARCH="arm64"
else
    ARCH="amd64"
fi

wget https://go.dev/dl/go1.20.1.linux-${ARCH}.tar.gz
sudo rm -rf ${GO_BASE_PATH}/go && sudo tar -C ${GO_BASE_PATH} -xf go1.20.1.linux-${ARCH}.tar.gz
rm go1.20.1.linux-${ARCH}.tar.gz
```